src/ddmd/attrib.d

```diff
@@ -55,6 +55,9 @@ extern (C++) abstract class AttribDeclaration : Dsymbol
 
     Dsymbols* include(Scope* sc, ScopeDsymbol sds)
```

Comment:
Also need to do the overrides of `include()`.

Reply:
Done